src/vendorcode/fsp/coffeelake/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "coffeelake"
```

Comment:
The typical convention is that raw bindgen wrapper crates have `-sys` appended to the crate names. It leaves room for a more idiomatic Rust interface to be exposed without the `-sys`. May I also suggest `fsp` in the crate name somewhere in case we ever want to publish this separately?

Reply:
Thanks!